libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSecurityLockout.h

```diff
@@ -148,25 +170,58 @@ typedef void (^SFPasscodeViewControllerPresentationBlock)(UIViewController*);
 + (void)removeDelegate:(id<SFSecurityLockoutDelegate>)delegate;
 
 
-/** Get the current lockout time, in seconds
+/**
+ Get the current lockout time, in seconds
+ @return The lockout time limit.
  */
 + (NSUInteger)lockoutTime;
 
 /**
  Gets the configured passcode length.
- @return The minimum passcode length.
+ @return The passcode length.
+ */
++ (NSUInteger)passcodeLength;
+
+/**
+ Gets the custom attribute from the connected app that enables biometric unlock.
+ @return Whether biometric unlock is enabled in the org.
+ */
++ (BOOL)biometricUnlockAllowed;
+
+/**
+ Whether the user has opted in to biometric unlock.
+ @return User has opted in to biometric unlock.
+ */
++ (BOOL)biometricUnlockEnabled;
```

Comment:
just like userDeclinedBiometricUnlock, can we rename this to userAcceptedBiometricUnlock

Reply:
I would rather not.  `biometricUnlockEnabled` says exactly what this flag is supposed to represent.  I purposefully named `userDeclinedBiometricUnlock` differently as to not confuse it with the opposite of biometricUnlockEnabled, because they are not equivalent or comparable.  I am open to renaming userDeclinedBiometricUnlock.